beacon-chain/node/node.go

```diff
@@ -200,6 +203,14 @@ func New(cliCtx *cli.Context, opts ...Option) (*BeaconNode, error) {
 	if err != nil {
 		return nil, err
 	}
+
+	log.Debugln("Starting Blob Storage")
+	blobStorage, err := filesystem.NewBlobStorage(beacon.BlobStoragePath)
```

Comment:
The initialization order of setting up the top-level context and calculating functional args makes it complicated to directly initialize the BlobStorage value as a functional arg. `BlobStoragePath` is a work-around for that complication, allowing the functional arg to set the path as a string, deferring actual initialization until the New method runs.